<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-KHNQ7JL" failures="1" tests="3" name="Test" time="13.474" errors="0" timestamp="2023-05-15T13:25:07 IST">
  <testcase classname="com.demo.testcases.LoginTest" name="login" time="5.196">
    <failure type="java.lang.AssertionError" message="expected [Jay1] but found [Jay]">
      <![CDATA[java.lang.AssertionError: expected [Jay1] but found [Jay]
at com.demo.testcases.LoginTest.login(LoginTest.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="com.demo.testcases.LoginTest" name="Logout" time="2.378"/>
  <testcase classname="com.demo.testcases.LoginTest" name="validate_error_message" time="3.152"/>
</testsuite> <!-- Test -->
